src/cljs/chronograph_web/pages/organization/events.cljs

```diff
@@ -0,0 +1,30 @@
+(ns chronograph-web.pages.organization.events
+  (:require [re-frame.core :as rf]
+            [chronograph-web.http :as http]))
+
+
+(def ^:private get-organization-uri
+  "/api/organizations")
+
+
+(defn get-organization
+  [route]
+  (http/GET (str get-organization-uri
+                 (get-in route [:route-params :slug]))
+            {:timeout 8000
```

Comment:
Unnecessary to set timeouts in two places.

Reply:
Eek, I was supposed to have removed that. Will drop it.

Reply:
Fixed.